ntbs-service/Pages/Notifications/OverviewPartials/_ClinicalDetailsOverviewPartial.cshtml

```diff
@@ -0,0 +1,98 @@
+@{
+    var notificationId = ViewData["NotificationId"];
+}
+
+<div class="notification-overview-type-and-edit-container">
+    <h3 class="notification-details-type"> Clinical details </h3>
+    <a href="./Edit/ClinicalDetails?id=@notificationId" class="notification-edit-link"> Edit </a>

```

Comment:
Can I suggest that we also add a class to it that means it doesn't change colour when visited?
See https://design-system.service.gov.uk/styles/typography/#links, and maybe double check with Sorcha that's what we want :)